libcontainer/intelrdt/intelrdt.go

```diff
@@ -161,16 +161,25 @@ type Manager interface {
 	Set(container *configs.Config) error
 }
 
+type GroupType string
+
+const (
+	MON      GroupType = "MON"
+	CTRL_MON GroupType = "CTRL_MON"
+)
+
 // This implements interface Manager
 type IntelRdtManager struct {
 	mu     sync.Mutex
 	Config *configs.Config
 	Id     string
 	Path   string
+	Type   GroupType
```

Comment:
Why `Type` cannot be just string? Is it necessary to introduce `GroupType` type?